osu.Game/Storyboards/Drawables/DrawableStoryboardVideo.cs

```diff
@@ -23,11 +21,21 @@ public DrawableStoryboardVideo(StoryboardVideo video)
         {
             Video = video;
 
-            RelativeSizeAxes = Axes.Both;
+            // In osu-stable, a mapper can add a scale command for a storyboard.
```

Comment:
accidentally a word?

```suggestion
            // In osu-stable, a mapper can add a scale command for a storyboard video.
```